Clarify inputs for functions (#156)
cfrg/draft-irtf-cfrg-voprf: draft-irtf-cfrg-voprf.md

element in the scalar field `GG` and pkX is the product of skX and the
group's fixed generator.

> For base mode, servers don't need the public key `pkS` produced by KeyGen.

Reviewer: ```suggestion
For base mode, servers do not need the public key `pkS` produced by KeyGen.
```

element in the scalar field `GG` and pkX is the product of skX and the
group's fixed generator.

> For unverifiable modes, servers don't need to make the resulting public key `pkS`
> accessible for clients.

Reviewer 2: ```suggestion
For base mode, servers don't need the public key `pkS` produced by KeyGen.
```

# Ciphersuites {#ciphersuites}

> A ciphersuite (also referred as 'suite' on this document) for the protocol

Reviewer: ```suggestion
A ciphersuite (also referred as 'suite' in this document) for the protocol
```

> A ciphersuite (also referred as 'suite' in this document) for the protocol

Reviewer: ```suggestion
A ciphersuite (also referred to as 'suite' in this document) for the protocol
```
